CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ option (DO_INCLUDE_PLUGINS "Build subproject plugins" OFF)
 option (DO_INCLUDE_SDK "Build subproject sdk-cpp" OFF)
 
 option (DO_BUILD_TESTS "Set DO_BUILD_TESTS to OFF to skip building tests." ON)
+option (DO_BUILD_FOR_SNAP "Enable DO Snap build option" ON)
```

Comment:
The default should be OFF otherwise all builds will start generating snaps.

Reply:
oh ok my bad, I misunderstood the meaning of that